lib/menu.rb

```diff
@@ -0,0 +1,16 @@
+class Menu
+  attr_reader :dishes
+
+  def initialize
+    @dishes = {
+      'bolognese' => 7.50,
+      'carbonara' => 6.50,
+      'puttanesca' => 5.50,
+      'arrabiata' => 4.50
+    }
+  end
```

Comment:
This works well - do you plan to implement an updateable menu so that the code can be re-used for other restaurants?